packages/core/src/App/Containers/Layout/header/traders-hub-home-button.tsx

```diff
@@ -31,11 +39,15 @@ const TradersHubHomeButton = observer(() => {
                     pathname === routes.traders_hub_v2 ||
                     pathname === routes.wallets,
             })}
-            onClick={() => history.push(redirect_routes())}
+            onClick={() => history.push(redirectRoutes())}
         >
             <div className='traders-hub-header__tradershub--home-logo'>
                 <Icon
-                    icon={is_dark_mode_on ? 'IcAppstoreHomeDark' : 'IcAppstoreTradersHubHome'}
+                    icon={classNames({
+                        [TradershubIcon.default]: !is_next_wallet_enabled && !is_dark_mode_on,
```

Comment:
shouldn't there be always only one class? looks kinda weird to have this construct (intented to support multiple classes) here